### PR TITLE
Make a copy of the eCR datastore as a parquet file

### LINKED ISSUE #210: Create a Parquet and CSV copies of the eCR datastore whenever the delta table is updated
**Description**

In order to make it easier for LAC county folks to begin working with data in the eCR data directly we should provide them the dataset in both Parquet and CSV format in addition to the Delta Table we are already creating.

**Acceptance Criteria**

GIVEN the Synapse notebook for updating the eCR datastore
WHEN the notebook is run
THEN the eCR datastore Delta Table is refreshed and a copies are exported as parquet and CSV files stored in Azure blob storage.

### PR DESCRIPTION
**Pull request by DanPaseltiner**
This PR modifies our `updateECRDataStore` notebook to all produce a copy of the entire datastore as a parquet file after updating the delta table with additional records.

fixes #210

### CODE REVIEW

**Comment by bamader** [comment]
@DanPaseltiner I can't tag you as a reviewer because you opened the PR initially, but def feel free to take a look and see if this matches what you were expecting. I won't merge until I hear from you.

File: scripts/Synapse/updateECRDataStore.ipynb [hunk 1]
```diff
@@ -283,7 +295,30 @@
...
+    "from azure.storage.filedatalake import DataLakeServiceClient\n",
+    "import time\n",
+    "\n",
+    "# Class to allow us to use the spark pool's managed identity to access blobs \n",
+    "# and containers (to move them around and rename them)\n",
+    "class spoof_token:\n",
+    "    def get_token(*args, **kwargs):\n",
+    "        return AccessToken(\n",
+    "            token=mssparkutils.credentials.getToken(audience=\"storage\"),\n",
+    "            expires_on=int(time.time())+60*10 # some random time in future... synapse doesn't document how to get the actual time\n",
```
Comment (m-goggins) [inline scripts/Synapse/updateECRDataStore.ipynb:320]: Have you run this in the pipeline? I believe this code works when it's run in a notebook but not in a pipeline. 

@DanPaseltiner 's fix to the service principal might help us get around the issue

Comment (bamader) [reply scripts/Synapse/updateECRDataStore.ipynb:320]: Ugh no I was just running it in the notebook 😭 . What's the issue with it in a pipeline?

Comment (DanPaseltiner) [reply scripts/Synapse/updateECRDataStore.ipynb:320]: Nice catch @m-goggins. I agree we should switch here to using our service principle. @bamader the short answer is that we have run into issues with getting the notebooks to run using a managed identity when they are run via the pipeline. To get around this we can use a service principle and get a credential object using its `client_id` and `client_secret`. 

[This PR](https://github.com/CDCgov/phdi/pull/752/files) add in functionality to the SDK to use our own credential manager class to get a credential object by providing a client ID and client secret. In the notebook we can read these values out of a key vault using the linked service that @m-goggins created. One of the notebooks currently deployed in `dev2` should have an example of reading from the key vault and using the service principle creds.

**Review by bamader — APPROVED** [review]

**Pull request merged**